inputs/blast_wave.pin

```diff
@@ -18,6 +18,8 @@
 
 <phoebus>
 problem = sedov
+ix1_bc = reflect
+ox1_bc = gr_outflow
```

Comment:
are these necessary up here still?

Reply:
For reasons that I don't fully understand, they seem to be. Otherwise it crashes that a BC is not registered. Likely an issue on the phoebus side.

Reply:
It's a phoebus issue. Found it below in my review. The switches in the phoebus BC registration need to be removed. Just register all BCs. :)